### PR TITLE
Add install script

### PR DESCRIPTION
**Pull request by fcollonval**
Add a script to run `jupyter labextension install .` to all extensions + a global script `install-ext`

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks!

Should we keep both `install` and `link`?

**Comment by fcollonval** [comment]
> Should we keep both `install` and `link`?

Following https://github.com/jupyterlab/extension-cookiecutter-ts/pull/67, I think it will be better to switch to `install` as `link` is for grafting extensions in core if I remembered correctly.

But of course first I need to fix this PR because `install` conflict with `npm install` :face_with_head_bandage:

**Comment by jtpio** [comment]
I wonder whether we should document the differences between `install` and `link` somewhere.

The JupyterLab documentation could be the appropriate place, but a quick word in this repo (for example in the hello world example) could also work.

There are some details in this discourse topic too: https://discourse.jupyter.org/t/about-jupyter-labextension-link-v-s-install/2201

**Comment by fcollonval** [comment]
I removed all usage of `link` and add a small note in `hello-world`

File: basics/hello-world/README.md [hunk 1]
```diff
@@ -166,7 +166,10 @@ components that you want to use in your project.
 
 The second step runs the build script. In this step, the TypeScript code gets
 converted to javascript using the compiler `tsc` and stored in a `lib`
-directory. Finally, the module is linked to JupyterLab.
+directory. Finally, the module is installed into JupyterLab.
+
+> There is also a `jupyter labextension link <path>` command. It can be used to
+> install packages but not considered them as extensions (more information in [that discussion](https://discourse.jupyter.org/t/about-jupyter-labextension-link-v-s-install))
```
Comment (jtpio) [inline basics/hello-world/README.md:172]:
```suggestion
> install packages but not consider them as extensions (more information in [that discussion](https://discourse.jupyter.org/t/about-jupyter-labextension-link-v-s-install))
```

**Comment by fcollonval** [comment]
Thanks @jtpio

**Pull request merged**